package.json

```diff
@@ -10,6 +10,10 @@
   },
   "author": "",
   "license": "ISC",
+  "files": [
+    "!test",
```

Comment:
This line isn't required - the test directory doesn't end with `.js`, so it's already excluded. The `*` isn't recursive - you'd need to use `**/*.js` to recursively include all `.js` files.

```suggestion
```